src/lib.rs

```diff
@@ -206,36 +252,37 @@ macro_rules! ct_eq_slice_gen {
             for i in 0..x_len {
                 flag |= x[i] ^ y[i];
             }
-            $eq(flag,0)
+            <$code as ConstantTime>::ct_eq(flag,0)
```

Comment:
Your comment https://github.com/valarauca/consistenttime/issues/5#issuecomment-272042450 indicates that maybe this line should be reverted to `$eq`, which means adding `$eq` back.